### PR TITLE
Added validations to post requests and error handling

### PR DESCRIPTION
**Pull request by mhz-edu**
Added validation of post requests and basic error handling

### CODE REVIEW

File: src/app.ts [hunk 1]
```diff
@@ -11,4 +11,16 @@ app.use('/api/users', usersRouter);
 app.use(imagesRouter);
 app.use('/api/albums', albumsRouter);
 
+const errorHandler: ErrorRequestHandler = (error, req, res, next) => {
+  const statusCode = error.statusCode || 500;
+  const data = error.data;
+  const message = error.message;
```
Comment (irenlian) [inline src/app.ts]: const { data, message, statusCode = 500 } = error;

File: src/app.ts [hunk 2]
```diff
@@ -11,4 +11,16 @@ app.use('/api/users', usersRouter);
...
+  res.status(statusCode).json({
+    message: message,
+    error: data,
+  });
+};
```
Comment (irenlian) [inline src/app.ts]: I'd anticipate seeing the handler in a separate file.

File: package.json [hunk 3]
```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "exif": "0.6.0",
     "express": "^4.17.2",
+    "express-validator": "^6.14.0",
```
Comment (irenlian) [inline package.json:26]: I believe this library is fine.
FYI, here is one more to look into. I met it in the most projects:
https://www.npmjs.com/package/joi

**Review by irenlian — APPROVED** [review]

File: src/app.ts (hunk 1 above)
Comment (mhz-edu) [reply src/app.ts]: updated

File: src/app.ts (hunk 2 above)
Comment (mhz-edu) [reply src/app.ts]: Moved to separate file

File: package.json (hunk 3 above)
Comment (mhz-edu) [reply package.json:26]: Thanks, going to try next time

**Pull request merged**